packages/squiggle-lang/src/public/SqValue/index.ts

```diff
@@ -308,6 +310,31 @@ export class SqDomainValue extends SqAbstractValue<"Domain", SqDomain> {
   }
 }
 
+export class SqBoxedValue extends SqAbstractValue<"Boxed", unknown> {
+  tag = "Boxed" as const;
+
+  get value() {
+    return wrapValue(this._value.value.value, this.context);
```

Comment:
```this._value.value.value``` is kind of silly, but I'm not sure what's better.

Reply:
Maybe we could pull the type out, and not used Boxed underneath this, given the simplicity. But that might be a pain later.